src/plugins/elasticsearch/lib/__tests__/check_es_version.js

```diff
@@ -73,14 +73,16 @@ describe('plugins/elasticsearch', function () {
       }, _.noop);
     });
 
-    it('passes if that single node is a client node', function () {
+    it('fails if that single node is a client node', function () {
       setNodes(
         '1.4.4',
         { version: '1.4.2', attributes: { client: 'true' } },
         '1.4.5'
       );
-
-      return checkEsVersion(server);
+      return checkEsVersion(server)
+      .then(function() {
+        throw new Error('expected validation to fail')
+      }, _.noop);
```

Comment:
Hey @tsullivan, I tried it the way you suggested but it didn't work. So I had Spencer look my work over and he made me realize why: `checkEsVersion` is asynchronous and returns a Promise so that assertion won't work as expected. However, I do agree that we can make a stronger assertion here than the one I'm currently making. I'll update this test shortly.